.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+
+python:
+      - "3.4"
+      - "3.5"
+      - "3.5-dev"
+      - "3.6"
+      - "3.6-dev"
+      - "nightly" # currently points to 3.7-dev
+install:
+    - "pip install -r requirements.txt"
```

Comment:
Bear in mind that `pip install -e .[dev]` already does this. I prefer the shorter method but won't argue much about it, however we should probably be consistent with `CONTRIBUTING.md` and avoid duplicating the places where we specify the dependencies.

Reply:
good catch! I'm removing it. 

And what about the requirements-test.txt?